docs/SYSTEM_OVERVIEW.md

```
@@ -140,13 +129,13 @@ Stores **decomposed scores** - 15 columns per post per epoch:
 Current and historical governance periods with weight distribution.
 
 ### governance_votes
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)